build/thirdparty/darwin/Dockerfile

```diff
@@ -85,7 +85,7 @@ ENV GOOS darwin
 ENV GOARCH amd64
 ENV DEST /usr/local/kubebuilder/bin/
 RUN mkdir -p $DEST || echo ""
-RUN git clone https://github.com/kubernetes-incubator/reference-docs $GOPATH/src/github.com/kubernetes-incubator/reference-docs --depth=1
+RUN git clone https://github.com/kubernetes-incubator/reference-docs $GOPATH/src/github.com/kubernetes-incubator/reference-docs --branch kubebuilder  --depth=1
```

Comment:
Are the changes in kubebuilder branch will ever be merged in master for that repo ?